cmd/aws-s3-proxy/pkg/serve.go

```diff
@@ -127,7 +127,7 @@ func initConfig() {
 	}
 
 	// Check for ENV variables set
-	// All ENV vars will be prefixed with "BENDER_"
+	// All ENV vars will be prefixed with "S3_PROXY_"
 	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
 	viper.SetEnvPrefix("s3-proxy")
```

Comment:
The prefix is `s3-proxy` with a dash but the comment says to use an underscore. Just want to verify that it is an underscore and not a dash.